polyfactory/value_generators/constrained_uuid.py

```diff
@@ -7,8 +7,8 @@
 def handle_constrained_uuid(uuid_version: Literal[1, 3, 4, 5], faker: Faker) -> UUID:
     if uuid_version == 1:
         return uuid1()
-    if uuid_version == 3:
+    if uuid_version == 3:  # TODO: investigate @guacs # noqa: PLR2004, FIX002
```

Comment:
@guacs I messaged you a potential fix, can implement if it works

Reply:
As for `ruff` complaining about magic values, I think we can just constants for the numbers instead. Something like:

```python

UUID_VERSION_1 = 1
UUID_VERSION_2 = 2
...

def handle_constrained_uuid(...) -> UUID:
    if uuid_version == UUID_VERSION_1:
        ...
```